src/xcode/ENA/TestPlans/UITests.xctestplan

```diff
@@ -133,7 +133,9 @@
         "ENAUITests_16_UniversalQRCodeScanner\/test_screenshot_UniversalQRCodeScannerAndQRScannerInfoScreen()",
         "ENAUITests_19_TicketValidation\/test_screenshot_ticketValidation_failed()",
         "ENAUITests_19_TicketValidation\/test_screenshot_ticketValidation_passed()",
-        "ENAUITests_20_CertificateReissuance"
+        "ENAUITests_20_CertificateReissuance",
+        "ENAUITests_21_FamilyMember\/test_screenshot_RegisterCoronaTestFromUniversalQRCodeScanner()",
+        "ENAUITests_21_FamilyMember\/test_screenshot_familyMemberViewOverviewDeleteAll()"
```

Comment:
if I open the test plan with Xcode, these two tests are NOT included in the UITests.xctestplan. Don't know why they are shown here.